go/interpreter/lfvm/opcode.go

```diff
@@ -218,179 +246,14 @@ const (
-	ISZERO:   "ISZERO",
-	SUB:      "SUB",
-	ADD:      "ADD",
-	DUP5:     "DUP5",
-	DUP1:     "DUP1",
-	EQ:       "EQ",
-	LT:       "LT",
-	SLT:      "SLT",
-	SHR:      "SHR",
-	DUP6:     "DUP6",
-	RETURN:   "RETURN",
-	REVERT:   "REVERT",
-	PUSH32:   "PUSH32",
-
-	PUSH0:  "PUSH0",
-	PUSH3:  "PUSH3",
-	PUSH5:  "PUSH5",
-	PUSH6:  "PUSH6",
-	PUSH7:  "PUSH7",
-	PUSH8:  "PUSH8",
-	PUSH9:  "PUSH9",
-	PUSH10: "PUSH10",
-	PUSH11: "PUSH11",
-	PUSH12: "PUSH12",
-	PUSH13: "PUSH13",
-	PUSH14: "PUSH14",
-	PUSH15: "PUSH15",
-	PUSH16: "PUSH16",
-	PUSH17: "PUSH17",
-	PUSH18: "PUSH18",
-	PUSH19: "PUSH19",
-	PUSH20: "PUSH20",
-	PUSH21: "PUSH21",
-	PUSH22: "PUSH22",
-	PUSH23: "PUSH23",
-	PUSH24: "PUSH24",
-	PUSH25: "PUSH25",
-	PUSH26: "PUSH26",
-	PUSH27: "PUSH27",
-	PUSH28: "PUSH28",
-	PUSH29: "PUSH29",
-	PUSH30: "PUSH30",
-	PUSH31: "PUSH31",
-	DUP7:   "DUP7",
-	DUP8:   "DUP8",
-	DUP9:   "DUP9",
-	DUP10:  "DUP10",
-	DUP11:  "DUP11",
-	DUP12:  "DUP12",
-	DUP13:  "DUP13",
-	DUP14:  "DUP14",
-	DUP15:  "DUP15",
-	DUP16:  "DUP16",
-	SWAP4:  "SWAP4",
-	SWAP5:  "SWAP5",
-	SWAP6:  "SWAP6",
-	SWAP7:  "SWAP7",
-	SWAP8:  "SWAP8",
-	SWAP9:  "SWAP9",
-	SWAP10: "SWAP10",
-	SWAP11: "SWAP11",
-	SWAP12: "SWAP12",
-	SWAP13: "SWAP13",
-	SWAP14: "SWAP14",
-	SWAP15: "SWAP15",
-	SWAP16: "SWAP16",
-
-	STOP: "STOP",
-	PC:   "PC",
-
-	MUL:        "MUL",
-	DIV:        "DIV",
-	SDIV:       "SDIV",
-	MOD:        "MOD",
-	SMOD:       "SMOD",
-	ADDMOD:     "ADDMOD",
-	MULMOD:     "MULMOD",
-	EXP:        "EXP",
-	SIGNEXTEND: "SIGNEXTEND",
-
-	SHA3: "SHA3",
-
-	SGT: "SGT",
-
-	OR:   "OR",
-	XOR:  "XOR",
-	NOT:  "NOT",
-	BYTE: "BYTE",
-	SHL:  "SHL",
-	SAR:  "SAR",
-
-	MSTORE:  "MSTORE",
-	MSTORE8: "MSTORE8",
-	MLOAD:   "MLOAD",
-	MSIZE:   "MSIZE",
-	MCOPY:   "MCOPY",
-
-	SLOAD:  "SLOAD",
-	SSTORE: "SSTORE",
-	TLOAD:  "TLOAD",
-	TSTORE: "TSTORE",
-
-	LOG0: "LOG0",
-	LOG1: "LOG1",
-	LOG2: "LOG2",
-	LOG3: "LOG3",
-	LOG4: "LOG4",
-
-	ADDRESS:        "ADDRESS",
-	BALANCE:        "BALANCE",
-	ORIGIN:         "ORIGIN",
-	CALLER:         "CALLER",
-	CALLVALUE:      "CALLVALUE",
-	CALLDATALOAD:   "CALLDATALOAD",
-	CALLDATASIZE:   "CALLDATASIZE",
-	CALLDATACOPY:   "CALLDATACOPY",
-	CODESIZE:       "CODESIZE",
-	CODECOPY:       "CODECOPY",
-	GASPRICE:       "GASPRICE",
-	EXTCODESIZE:    "EXTCODESIZE",
-	EXTCODECOPY:    "EXTCODECOPY",
-	RETURNDATASIZE: "RETURNDATASIZE",
-	RETURNDATACOPY: "RETURNDATACOPY",
-	EXTCODEHASH:    "EXTCODEHASH",
-	CREATE:         "CREATE",
-	CALL:           "CALL",
-	CALLCODE:       "CALLCODE",
-	DELEGATECALL:   "DELEGATECALL",
-	CREATE2:        "CREATE2",
-	STATICCALL:     "STATICCALL",
-	SELFDESTRUCT:   "SELFDESTRUCT",
-
-	BLOCKHASH:   "BLOCKHASH",
-	COINBASE:    "COINBASE",
-	TIMESTAMP:   "TIMESTAMP",
-	NUMBER:      "NUMBER",
-	PREVRANDAO:  "PREVRANDAO",
-	GAS:         "GAS",
-	GASLIMIT:    "GASLIMIT",
-	CHAINID:     "CHAINID",
-	SELFBALANCE: "SELFBALANCE",
-	BASEFEE:     "BASEFEE",
-	BLOBHASH:    "BLOBHASH",
-	BLOBBASEFEE: "BLOBBASEFEE",
-
+var to_string = map[OpCode]string{
```

Comment:
nit: our code style convention would require this to be a `toString` or `opCodeStrings`; we could also switch to the generator based `stringer` solution in a follow-up PR.